frontend/packages/gitops-plugin/src/components/details/GitOpsResourcesSection.tsx

```diff
@@ -0,0 +1,158 @@
+import * as React from 'react';
+import {
+  Stack,
+  StackItem,
+  Split,
+  SplitItem,
+  Card,
+  CardBody,
+  Tooltip,
+} from '@patternfly/react-core';
+import { HeartBrokenIcon } from '@patternfly/react-icons';
+import { useTranslation } from 'react-i18next';
+import { ResourceIcon } from '@console/internal/components/utils';
+import { GitOpsEnvironmentService, GitOpsHealthResources } from '../utils/gitops-types';
+import './GitOpsResourcesSection.scss';
+
+interface GitOpsResourcesSectionProps {
+  services: GitOpsEnvironmentService[];
+  secrets: GitOpsHealthResources[];
+  deployments: GitOpsHealthResources[];
+  routes: GitOpsHealthResources[];
+  roleBindings: GitOpsHealthResources[];
+  clusterRoles: GitOpsHealthResources[];
+  clusterRoleBindings: GitOpsHealthResources[];
+}
+
+const getUnhealthyResources = () => (acc: string[], current: GitOpsHealthResources): string[] =>
+  current.health === 'Degraded' ||
+  current.health === 'Progressing' ||
+  current.health === 'Missing' ||
+  current.health === 'Unknown'
```

Comment:
Not sure if you want to create an enum for these strings so they can be more easily referenced / tracked.

Reply:
Added to [GITOPS-1283](https://issues.redhat.com/browse/GITOPS-1283)